jjb/dynamic/scale-ci_baseline.yml

```diff
@@ -14,12 +14,15 @@
         fi
         # Re-enable logging
         set -x
-        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
-        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
-        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
-        export PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
-        export PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
-        chmod 600 ${PRIVATE_KEY}
+        git clone https://${SSHKEY_TOKEN}@${SSHKEY_REPO}
+        if [ -f ${WORKSPACE}/${SSHKEY_REPOPATH_PUB} ] && [ -f ${WORKSPACE}/${SSHKEY_REPOPATH_PRIV} ]
```

Comment:
We need to update other jobs to do the same as they use the same logic to fetch the ssh keys.

Reply:
This can be tackled in a separate PR I guess.